radixdlt/src/main/java/org/radix/GenerateNodeKey.java

```diff
@@ -17,45 +17,36 @@
 
```

Comment:
This file should be deleted.  It was for a specific purpose to generate keys in specific shards for 1mtps project, and is no longer required.

Reply:
Done